tasks/71_binary-encode/binary-encode.js

```diff
@@ -0,0 +1,8 @@
+export function binaryEncode(text) {
+  if (text === undefined) {
+    return '';
+  }
```

Comment:
Still readable, but less code:
```suggestion
export function binaryEncode(text = '') {
```